microcosm_flask/swagger/definitions.py

```diff
@@ -65,22 +73,43 @@ def build_swagger(graph, ns, operations):
     )
     add_paths(schema.paths, base_path, operations)
     add_definitions(schema.definitions, operations)
-    try:
-        schema.validate()
-    except Exception:
-        logger.exception("Swagger definition did not validate against swagger schema")
-        raise
+
+    if validate_schema:
```

Comment:
This will mean any service using this lib will need to add another test to validate their specific schema.